docs/AreaTable.md

```diff
@@ -2296,7 +2296,7 @@ This dbc contains the zone and subzone lists. For the purposes of this wiki arti
 | 4629     | Winter's Heart                                       | 571   | 67     |
 | 4630     | The North Sea                                        | 571   | 0      |
 | 4631     | The Broodmother's Nest                               | 571   | 67     |
-| 4632     | Dalaran Floating Rocks                               | 571   | 4395   |
+| 4632     | Dalaran FLOATing Rocks                               | 571   | 4395   |
```

Comment:
:S

Reply:
Yea still reviewing it, I got 75% of it done today so only a few files left for me to double check